src/gurt/dlog.c

```diff
@@ -631,6 +672,15 @@ void d_vlog(int flags, const char *fmt, va_list ap)
 		clog_unlock();
 		return;
 	}
+	/* 
+	 * If logging an error or higher, increment the error counter for
+	 * the facility. This can be used by monitoring solutions to detect
+	 * increased error rates.
+	 * 
+	 * TODO: Count by errno too? Might be too expensive.
```

Comment:
It wouldn't be possible to count by error no just yet.  I have in mind to re-write the logging macros to take "rc" as a parameter but for now logging is just printf so extracting the errno won't be possible.   Re-writing every logging call (or at least D_ERROR + D_WARN) is a big operation however so I want to get it right.

I've started on https://github.com/daos-stack/daos/pull/12575 which is the first step to allow us to make the change programatically but it's early days.  Once we've defined the new interface and got the tools then we've got the task of migrating the code and that'll be disruptive - hence the desire to do it slowly and correctly the first time.